sdk/eventhub/Azure.Messaging.EventHubs/tests/Primitives/EventProcessorTests.StartStop.cs

```diff
@@ -1038,52 +1042,6 @@ public async Task StopProcessingIsSafeToCallAfterStopping(bool async)
             Assert.That(GetRunningProcessorTask(mockProcessor.Object), Is.Null, "There should be no active task for processing.");
         }
 
-        /// <summary>
-        ///   Verifies functionality of the <see cref="EventProcessor{TPartition}.StopProcessing" />
-        ///   method.
-        /// </summary>
-        ///
-        [Test]
-        [TestCase(true)]
-        [TestCase(false)]
-        public async Task StopProcessingSurfacesExceptions(bool async)
```

Comment:
This is no longer a valid scenario, as the task captures and surfaces its own errors.   With the added try/catch protection, the task will not fault and will retry instead.